tests/Exception/ClientTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Chromatic\OrangeDam\Http;
+
+use Chromatic\OrangeDam\Exceptions\InvalidArgumentException;
+use Chromatic\OrangeDam\Exceptions\OrangeDamException;
+use GuzzleHttp\Client as GuzzleClient;
+use PHPUnit\Framework\TestCase;
+
+final class ClientTest extends TestCase
+{
+    public function testClientCreation(): void
```

Comment:
Can we add some comments that explain what we are testing and why?

Reply:
Updated.